pkg/apis/v1alpha5/machine_disruption_gate_status.go

```diff
@@ -0,0 +1,45 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha5
+
+import (
+	"knative.dev/pkg/apis"
+)
+
+// MachineDisruptionGateStatus defines the observed state of MachineDisruptionGate
+type MachineDisruptionGateStatus struct {
+	// NextActiveTime is the next time the MachineDisruptionGate will be active.
+	// Will return the current time if it is currently active.
```

Comment:
Doesn't this mean it'll be reconciling something fierce? Maybe we can just have a boolean?

Reply:
Doesn't have to be fierce. I haven't implemented this yet, but we can easily just compute the next crontab hit, and re-reconcile when we get there.